databricks/koalas/tests/test_series.py

```diff
@@ -2226,3 +2227,177 @@ def test_backfill(self):
             # Test `inplace=True`
             kser.backfill(inplace=True)
             self.assert_eq(expected, kser)
+
+    # TODO: Remove skip after dropping Python 3.5
+    @unittest.skipIf(
+        sys.version_info.minor <= 5,
+        "The columns order is not preserved in the result in Python<=3.5",
```

Comment:
Why is the result column order not preserved in Python 3.5?

Reply:
Oh, It was test problem with `OrderedDict`.

Fixed test, thanks.